app/models/imaage.rb

```diff
@@ -0,0 +1,5 @@
+class Imaage < ApplicationRecord
+  
+  belongs_to :albums
```

Comment:
when of the errors  I got stuck with was (album must exist) when I was instantiating an @imaage, so this was the solution for it, to pluralize the album in the Imaage model, I didn't think of that the first place cuz I used to have the model whom the current belongs_to without an s